src/element/bounds.ts

```diff
@@ -406,3 +406,10 @@ export const getClosestElementBounds = (
 
   return getElementBounds(closestElement);
 };
+
+export const getCommonBoundingBox = (
```

Comment:
I thought we must already have such a util, and lo and behold we already have 2 of them (slightly different). One in `src/disitrubte.ts` and another in `src/align.ts`.

Reply:
Correction, we have 3 (so now 4). There's also `getCommonBounds` in `src/element/bounds.ts` :smile:

Reply:
using the getCommonBounds directly!